docs/reference/contrib/settings_global.rst

```diff
@@ -0,0 +1,305 @@
+.. _settings_global:
```

Comment:
passing comment. we now prefer docs to be written in markdown.

You can use `{eval-rst}` (ref: https://myst-parser.readthedocs.io/en/latest/sphinx/use.html) for anything that is RST specific, should you need it